responses.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Where did we land on putting this file under something like `lib/responses.json`?

Reply:
I'll move it, but it's now weird that out localisation file is in `lib/` :troll:

Reply:
Fair. What if we put our strings in `i18n/`?